neuraxle/api/flask.py

```diff
@@ -145,3 +148,43 @@ def get(self):
         )
 
         return app
+
+
+class RestAPICaller(NonFittableMixin, BaseStep):
```

Comment:
This class should be like the rest api server: inherit from pipeline and accept an encoder and a decoder objects for the json.

Reply:
this cannot be done. not feasible. I have two rest api routes and that would not work that way.